modules/ROOT/pages/rest-sdk/edits/project-setup.adoc

```diff
@@ -0,0 +1,62 @@
+= Setup and Installation
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+To get started with REST SDK, ensure you meet the prerequisites and then follow the
+steps in this section to install the REST SDK plugin.
```

Comment:
```suggestion
steps to install the REST SDK plugin.
```
Per style ref., we try to avoid self-referential locators like "in this section" or "in the following topic" or similar so that the content is reusable.